common/monitoring_api/src/lib.rs

```diff
@@ -62,8 +62,6 @@ pub struct MonitoringHttpClient {
     client: reqwest::Client,
     /// Path to the hot database. Required for getting db size metrics
     db_path: Option<PathBuf>,
-    /// Path to the freezer database.
-    freezer_db_path: Option<PathBuf>,
```

Comment:
The freezer_db_path not being used looks like a bug on the monitoring api. 
https://github.com/sigp/lighthouse/blob/fff01b24ddedcd54486e374460855ca20d3dd232/common/monitoring_api/src/lib.rs#L131-L133

It should fetch `self.freezer_db_path` here. My bad.

Reply:
Not sure if we should make a separate PR to fix this bug or fix it here itself.

Reply:
ahhh nice! I can fix it here